django_loci/apps.py

```diff
@@ -15,11 +23,21 @@ def __setmodels__(self):
         from .models import Location
         self.location_model = Location
 
+    def test_geocode(self):
+        location = GeoCodeView.geocode('Red Square')
+        try:
+            assert round(location.latitude) == 56 and round(location.longitude) == 38
+        except AssertionError:
```

Comment:
I would expect it to fail for other reasons, not `AssertionError`. I would expect it to fail for a network issue, the API key is incorrect, things like that. Have you tried to disconnect your laptop from the internet and run this and see what happens?

Reply:
`AssertionError` will be raised every time if geocodeing doesn't work including network issues and incorrect API key